### PR TITLE
Fixed login

### PR DESCRIPTION
**Pull request by rrpawar96**
This PR fixes the issue of Login, instead of sending parameters from headers it sends from JSON body: addresses- [FINERACT-726](https://issues.apache.org/jira/browse/FINERACT-726)  

This PR also resolves the PR of redirecting issue, after successful login now it redirects to the dashboard.

This PR also resolves the issue where if we refresh one tab then data of other tabs also getting refreshed, so it will stop that

### CODE REVIEW

**Comment by rrpawar96** [comment]
@avikganguly01, Please review and merge this PR.

**Comment by francisguchie** [comment]
@avikganguly01 @rrpawar96 I am interested in testing this

File: app/app.constants.js [hunk 1]
```diff
@@ -11,7 +11,10 @@
 		notAuthenticated: 'auth-not-authenticated'
 	})
 
-	.constant("TENANT_IDENTIFIER", "mobile")
+	.constant("TENANT_IDENTIFIER", "default")
+        .constant("Fineract-Platform-TenantId", "default")
+        .constant("tenantIdentifier", "default")
```
Comment (avikganguly01) [inline app/app.constants.js]: What is the purpose of defining the tenantIdentifier 3 times here?

File: app/src/authentication/authentication.service.js [hunk 2]
```diff
@@ -11,16 +11,16 @@
             isAuthenticated = false;
 
         // Set Access Token to requests
-        var setAccessToken = function (token) {
+       /* var setAccessToken = function (token) {
```
Comment (avikganguly01) [inline app/src/authentication/authentication.service.js]: Can these comments be removed if it's no longer relevant?

**Review by avikganguly01 — CHANGES_REQUESTED** [review]

**Comment by rrpawar96** [comment]
@avikganguly01, Thanks for the review, Comments Addressed, Please re-review.

**Comment by francisguchie** [comment]
@avikganguly01 
would this be merged soon?

**Comment by francisguchie** [comment]
@avikganguly01 @vishwasbabu can this be reviewed please and merged

**Review by avikganguly01 — APPROVED** [review]

**Pull request merged**